aws-throwaway/src/backend/cli/mod.rs

```diff
@@ -578,8 +579,19 @@ impl Aws {
             "AvailabilityZone={AZ},GroupName={}",
             self.placement_group_name
         );
+        // Secondary interfaces should not be used until they are configured.
+        let mut bring_down_secondary_interfaces = String::new();
+        for i in 1..definition.network_interface_count {
+            writeln!(
+                bring_down_secondary_interfaces,
+                "sudo ip link set dev ens{} down",
```

Comment:
What I don't like at all, is that you are bringing the machine interfaces down. If I were a user and something would shut down my interfaces I would be annoyed at the minimum.

Reply:
This is run on instance boot before the user has a chance to touch it so I figured it was fine.
Thanks for the feedback though, in the future I'll look to replace this with a solution that keeps the instances up but configures them so that they dont interfere.
For future reference I did only hit this issue on ubuntu 20.04.
22.04 seemed fine but I wasnt sure if 22.04 was actually handling it properly or just getting lucky.